lnbits/helpers.py

```diff
@@ -50,6 +50,9 @@ def template_renderer(additional_folders: Optional[List] = None) -> Jinja2Templa
     t.env.globals["SITE_DESCRIPTION"] = settings.lnbits_site_description
     t.env.globals["LNBITS_THEME_OPTIONS"] = settings.lnbits_theme_options
     t.env.globals["LNBITS_VERSION"] = settings.version
+    t.env.globals["LNBITS_CREATE_ACCOUNT"] = (
+        settings.lnbits_create_account and len(settings.lnbits_allowed_users) == 0
```

Comment:
- use `settings. allow_new_user_creation` property

Reply:
addressed in b9e6b915c477abdf5d3267ac4a5fa3b221c0ffaf